src/dev/bunHMRDevServer.ts

```
@@ -0,0 +1,794 @@
+  private async handleRequest(req: Request): Promise<Response> {
+    const url = new URL(req.url);
+    const pathname = url.pathname;
+  
+    try {
+      // Route to appropriate handler based on pathname
+      switch (pathname) {
+        case '/':
+        case '/html':
+          // HTML is simple - just serve the file and inject the HMR script
+          const htmlFile = await handleHTMLPageRequest('./example/build/html/pages/HtmlExample.html');
+          const htmlContent = await htmlFile.text();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)